README.md

```diff
@@ -62,3 +62,8 @@ new AMI to the AutoScaling groups, so that new EC2 instances will use the new AM
 Packer and cloudformation won't automatically remove old AMI versions, so this has to be
 done manually. AMIs can be deleted from the EC2 console by deregestering the AMI and
 deleting the related EBS snapshot.
+
+## SSHing into instances
+
+SSH into ElasticBeanstalk instances with the user `ec2-user`.
```

Comment:
Is it worth mentioning where we can find the passwords?  Or too secret for here?